src/main/java/edu/kis/vh/nursery/list/IntLinkedList.java

```diff
@@ -1,11 +1,11 @@
 package edu.kis.vh.nursery.list;
 
-public class IntLinkedList {
+public class IntLinkedList implements ListInterface {
 
 	Node last;
```

Comment:
Should be private.
Lack of encapsulation in Node class (Task 3 in "accessory i hermetyzacja")